patient-db/src/test/java/com/hartwig/hmftools/patientdb/clinical/readers/RunsJsonReaderTest.java

```diff
@@ -0,0 +1,44 @@
+package com.hartwig.hmftools.patientdb.clinical.readers;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.util.List;
+
+import com.google.common.io.Resources;
+import com.hartwig.hmftools.common.runcontext.RunContext;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+
+public class RunsJsonReaderTest {
+
+    @SuppressWarnings("UnstableApiUsage")
+    private static final String JSON_DIR = Resources.getResource("context" + File.separator + "RunJson").getPath();
+
+    @Test(expected = IllegalArgumentException.class)
+    public void throwsIllegalArgumentIfFileDoesntExist() {
+        RunsJsonReader.extractRunContexts(new File("/does/not/exist"));
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void throwsIllegalArgumentOnEmptyFile() {
+        RunsJsonReader.extractRunContexts(testJson("empty.json"));
+    }
+
+    @Test
+    public void throwsIllegalArgumentOnUnparseableFile() {
+        RunsJsonReader.extractRunContexts(testJson("missing_fields.json"));
+    }
```

Comment:
This looks to be missing the `expected =` attribute. And I don't see the JSON file?

Reply:
meant to remove this test